### DISCUSSION TITLE
BUG: Max file size profile picture

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#480] [General] [answered]
Found when editing profile page and trying to update profile picture

![image](https://user-images.githubusercontent.com/64191883/144749452-a60acfe8-45b6-4dc0-90ac-985cfd72845b.png)
 I'm good at maths but i'm still unsure how big this is. Maybe put 1MB ? (if thats the right amount)

**Comment by techsmyth** [comment, marked as answer]
Hi Rosa, 

yes the whole area of image management is in need of a refresh. I have put up an epic for this: https://github.com/alkem-io/alkemio/issues/486

Probably needs to be pretty high up the backlog...feel free to add ideas / requirements directly in there.

Cheers, Neil

**Comment by techsmyth** [comment]
If you have a few minutes I suggest you have a look at the new avatar selector ;)
